webapi/src/main/twirl/queries/sparql/v2/isEntityUsed.scala.txt

```diff
@@ -20,17 +20,19 @@
 @import org.knora.webapi.util.SmartIri
 
 @*
- * Checks whether an ontology entity is used (i.e. is the object
- * of any statements).
+ * Checks whether an entity is used (i.e. is the object of any statements).
  *
  * @param triplestore the name of the triplestore being used.
- * @param entityIri the IRI of the ontology.
+ * @param entityIri the IRI of the entity.
  * @param ignoreKnoraConstraints if true, knora-base:subjectClassConstraint and knora-base:objectClassConstraint will be ignored.
  * This is necessary when modifying the cardinalities of a class.
+ * @param ignoreRdfSubjectAndObject if true, rdf:subject and rdf:object will be ignored. This is necessary when checking
```

Comment:
So this ignores reifications (link values) referring to the IRI of the resource to be deleted if set to true? 

Wouldn't this mean that  a resource is referenced and hence cannot be deleted?

Reply:
I would expect that the IRI can be used as a rdf:subject (outgoing link) but not as an rdf:object.

Reply:
Suppose we want to erase resource `A`, and resource `B` has a link value that points to `A`, and the link value is marked as deleted. The link value will contain an `rdf:object` pointing to `A`. It's OK to ignore it, because `B` no longer refers to `A`.

Reply:
But the link value would have an `rdf:object` with the IRI of `A` that does not occur anymore as a subject in the triplestore, right?

Reply:
@tobiasschweizer 

> But the link value would have an `rdf:object` with the IRI of `A` that does not occur anymore as a subject in the triplestore, right?

That's correct. This wouldn't be a problem for Knora or for the GraphDB consistency checking. @lrosenth are you saying that these (marked-as-deleted) references would need to be erased for legal reasons?